### PR TITLE
83 context provider

### LINKED ISSUE #83: Integrate the descriptionProvider for cloudModeler in camunda-bpmn-js exposed cloud modeler
__What should we do?__

Integrate the descriptionProvider for cloudModeler in camunda-bpmn-js exposed cloud modeler

__Why should we do it?__

Having descriptions should be the default case

### PR DESCRIPTION
**Pull request by MaxTru**
closes #83

### CODE REVIEW

**Comment by MaxTru** [comment]
Ups, this is failing on Mac.... Need to investigate

**Comment by MaxTru** [comment]
That was a flaky test - will create seperate issue for that and ignore in this issue

File: test/camunda-cloud/ModelerSpec.js [hunk 1]
```diff
@@ -143,4 +143,19 @@ describe('<CamundaCloudModeler>', function() {
...
+  it('should inject descriptionContextProvider', function() {
+
+    // when
+    return createModeler(simpleXml).then(function(result) {
+
+      var modeler = result.modeler,
+          propertiesPanel = modeler.get('propertiesPanel');
+
+      // then
+      expect(propertiesPanel._descriptionConfig).to.exist;
```
Comment (barmac) [inline test/camunda-cloud/ModelerSpec.js:156]: Is there any way to test this without referring to private properties?

Comment (MaxTru) [reply test/camunda-cloud/ModelerSpec.js:156]: afaik not directly. We have the [BpmnPropertiesPanelRenderer](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/render/BpmnPropertiesPanelRenderer.js) at hand here, which does not offer ways to get its config

**Review by barmac — APPROVED** [review]
Let's merge it.

**Pull request merged**